core/model/admin.py

```diff
@@ -53,13 +61,18 @@ def has_password(self, password):
         return self.password_hashed == bcrypt.hashpw(password, self.password_hashed)
 
     @classmethod
-    def authenticate(cls, _db, email, password):
+    def authenticate(cls, _db, email: str, password: str):
         """Finds an authenticated Admin by email and password
         :return: Admin or None
         """
 
         def lookup_hook():
-            return get_one(_db, Admin, email=str(email)), False
+            return (
+                _db.query(Admin)
+                .filter(func.lower(Admin.email) == email.lower())
```

Comment:
It _might_ be worth uppercasing for comparisons, if a case folding function isn't available. There are characters out there that will map from uppercase to lowercase in surprising ways:

https://stackoverflow.com/questions/67106784/why-is-upper-casing-not-enough-for-case-insensitive-comparison

It's slightly safer to uppercase and compare than to lowercase and compare.

Reply:
It's a marginal (and unknown) amount of safer, but still a valid thought. Updated.